runtime/port/module.xml

```diff
@@ -108,35 +112,11 @@
 			<include-if condition="spec.linux_ztpf.*"/>
 			<exclude-if condition="spec.linux_x86.*"/>
 			<exclude-if condition="spec.win_x86.*"/>
-		</object>	
+		</object>
 		<object name="j9ri">
 			<include-if condition="spec.flags.port_runtimeInstrumentation"/>
 		</object>
-		<object name="j9ri_authorize_31">
```

Comment:
It seems the jit_runtimeInstrumentation flag should be set to false since files required for it to work are not included. Was that tried?

Reply:
Will double-check this flag.

Reply:
It fails in JIT after turning off these two flags:

CCN0793(I) Compilation failed for file /team/cjin/zos_openj9_build/test3/check/openj9_v1/runtime/compiler/../compiler/trj9/z/codegen/J9TreeEvaluator.cpp.  Object file not created.
/team/cjin/zos_openj9_build/test3/check/openj9_v1/runtime/compiler/../compiler/trj9/build/rules/zos-xlc/common.mk:99: recipe for target '/team/cjin/zos_openj9_build/test3/check/openj9_v1/runtime/compiler/../objs/compiler/trj9/z/codegen/J9TreeEvaluator.o' failed
make[3]: *** [/team/cjin/zos_openj9_build/test3/check/openj9_v1/runtime/compiler/../objs/compiler/trj9/z/codegen/J9TreeEvaluator.o] Error 12
make[3]: *** Waiting for unfinished jobs....
make[3]: Leaving directory '/team/cjin/zos_openj9_build/test3/check/openj9_v1/runtime/compiler/trj9'
Makefile:83: recipe for target 'trj9' failed
make[2]: *** [trj9] Error 2
make[2]: Leaving directory '/team/cjin/zos_openj9_build/test3/check/openj9_v1/runtime/compiler/trj9'
makefile:57: recipe for target 'default' failed
make[1]: *** [default] Error 2
make[1]: Leaving directory '/team/cjin/zos_openj9_build/test3/check/openj9_v1/runtime/compiler'
makefile:1138: recipe for target 'j9jitlauncher' failed
make: *** [j9jitlauncher] Error 2

Not sure whether something else needs to be turned off

Reply:
#1018 is opened to address this.